releasenotes/notes/seniority-zero-transformer-5d36d9658b46db8f.yaml

```diff
@@ -0,0 +1,4 @@
+features:
+  - |
+    Add SeniorityZeroTransformer to solve a chemistry problem in second quantization
+    with the restricted Hartree-Fock formalism.
```

Comment:
Maybe we can add a short usage example here

Reply:
Also saying it 'solves a chemistry problem` seems perhaps a bit of a mismatch. Saying it transforms a problem, and being a bit more specific about the nature of the problems its applies too, to make computation/solving a chemistry problem simpler/smaller/easier, or whatever the advantages of using it are, would be preferable I think. Then the example would follow on from there.